### PR TITLE
Add documentation for image-detector

### PR DESCRIPTION
**Pull request by KacperMalachowski**
**Description**

Changes proposed in this pull request:

- Add documentation for image-detector

**Related issue(s)**

### CODE REVIEW

File: development/image-detector/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+* Extract image urls from various file types
+* Update list of image urls in security scanners config file
+
+## Usage
+
+```
+Usage of image-detector:
+  --prow-config string
+    path to the prow config file (Required)
```
Comment (IwonaLanger) [inline development/image-detector/README.md:15]:
```suggestion
    path to the Prow config file (Required)
```

File: development/image-detector/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  --prow-jobs-dir string
+    path to the directory which contains prow job files (Required)
```
Comment (IwonaLanger) [inline development/image-detector/README.md:17]:
```suggestion
    path to the directory which contains Prow job files (Required)
```

File: development/image-detector/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  --terraform-dir string
+    path to the directory containing terraform files (Required)
```
Comment (IwonaLanger) [inline development/image-detector/README.md:19]:
```suggestion
    path to the directory containing Terraform files (Required)
```

File: development/image-detector/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  --sec-scanner-config
+    path to the security scanner config field (Required)
+  --kubernetes-dir string
+    path to the directory containing kubernetes deployments (Required)
```
Comment (IwonaLanger) [inline development/image-detector/README.md:23]:
```suggestion
    path to the directory containing Kubernetes deployments (Required)
```

File: development/image-detector/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  --tekton-catalog string
+    path to the tekton catalog directory (Required)
```
Comment (IwonaLanger) [inline development/image-detector/README.md:25]:
```suggestion
    path to the Tekton catalog directory (Required)
```

File: development/image-detector/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# image-detector
+
+Image Detector is a tool used to update security scanner config with list of images that lives in our prow cluster. To achive that it receives paths to files used to deploy prow or it's components.
```
Comment (IwonaLanger) [inline development/image-detector/README.md:3]: I've changed `our` to `your`, as the latter form is used as a general possessive adjective. You didn't use `our` meaning `Neighbors'`, right?

Comment (KacperMalachowski) [reply development/image-detector/README.md:3]: Not `Neighbors`, but 'Kyma'.

WOuld be better to use something like: `list of images that lives in prow cluster`?

File: development/image-detector/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+### Key Features:
+
+* Extract image urls from various file types
+* Update list of image urls in security scanners config file
```
[the remark this answers is not in the document]
Comment (KacperMalachowski) [reply development/image-detector/README.md:8]: 1. Yes, it specifies image detector features.
2. You're right.
3. I change that to singular. I want to have consistency with config file name and now spotted it's named `sec-scanner-config.yml` - singualr.

File: development/image-detector/README.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+# Image Detector
+
+## Overview
+
+Image Detector is a tool for updating the security scanner config with the list of images in Prow cluster. To achieve that, it receives paths to files used to deploy Prow or its components.
```
Comment (IwonaLanger) [inline development/image-detector/README.md:5]:
```suggestion
Image Detector is a tool for updating the security scanner config with the list of images in the Prow cluster. To achieve that, it receives paths to files used to deploy Prow or its components.
```

File: development/image-detector/README.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+## Key features:
+
+Image Detector:
+* Extracts image URLs from various file types
+* Updates the list of image URLs in the security scanners config file
```
Comment (IwonaLanger) [inline development/image-detector/README.md:11]:
```suggestion
* Updates the list of image URLs in the security scanner config file
```

**Review by IwonaLanger — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**